.github/workflows/ci.yml

```diff
@@ -112,3 +112,4 @@ jobs:
           ServiceControl_LicenseText: ${{ secrets.LICENSETEXT }}
           ServiceControl_Audit_LicenseText: ${{ secrets.LICENSETEXT }}
           Monitoring_LicenseText: ${{ secrets.LICENSETEXT }}
+          AZURE_ACI_CREDENTIALS: ${{ secrets.AZURE_ACI_CREDENTIALS }}
```

Comment:
This is required for the tests

Reply:
Can you why this is needed? Are you using managed identity? I have been doing countless improvements to support managed identity across the board and might be able to hint you in another direction.

Reply:
sure

Reply:
using it in https://github.com/Particular/ServiceControl/pull/4243/files#diff-fdc4bcf670eb0d22639787da65c10ff61c22dd4717e559595baaeb224425228fR206